src/i18n/en/secureMenu.ts

```diff
@@ -32,12 +32,30 @@ export default [
     {
         text: 'Advanced configurations', header: true, type: 'learn', key: 'secureAdvancedConfig', items: [
             { text: 'Manage bots', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/manage-bots/', key: 'manageBots' },
-            { text: 'Create WAF rule set', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/create-waf-rule-set/', key: 'wafRuleSet' },
-            { text: 'Tune WAF', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/tune-waf/', key: 'tuneWaf' },
             { text: 'Block Tor networks', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/block-tor-networks/', key: 'blockTor' },
         ]
     },
 
+    {
+        text: 'Create WAF rule set', header: true, type: 'learn', key: 'createRuleSet', slug: '/documentation/products/guides/secure/create-waf-rule-set/', hasLabel: 'menu.waf'
+    },
```

Comment:
Remove advanced configurations as a level 2 entry and keep only one list (connecting do create rule set above). WAF would be advanced for edge firewall, but check mode, tune , etc. are not advanced for WAF.